db/db.js

```diff
@@ -259,7 +253,7 @@ function getUserByGoogleId(profileId) {
         } else {
             return null;
         }
-    }).catch(err => perror(err));
+    }).catch(err => {throw new Error(err.message)});
```

Comment:
Who's catching these errors that are thrown?

Reply:
throw error when accessing db, let caller catch errors. By doing so, we may have an easier time for logging (no need of passing userInfo around all the time)